packages/node/src/utils/config.spec.ts

```diff
@@ -118,6 +118,32 @@ describe('getBaseWebpackPartial', () => {
     });
   });
 
+  describe('the additionalEntries option', () => {
+    it('should have multiple entries', () => {
+      const result = getBaseWebpackPartial({
+        ...input,
+        additionalEntryPoints: [
+          {
```

Comment:
Also up for debate but i origanlly decided on an Object because 
{ 
  scriptname: 'path.ts',
  anotherscriptname: 'path2.ts'
}
adds a lot less overhead to the workspace

Reply:
and it can be parsed out as key value pairs easily enough